### ISSUE TITLE
[ASK] Scan using button or continous mode

### ISSUE THREAD

**Issue by felixcary** [luis901101/honeywell_scanner#5]
I was looking at Honeywell Demo Apps and they have some button, 'Scan' button and 'Continous' mode, is there any function that I could use for manually scan (not using hardware button) or maybe 'Continous' mode scan? thank you
![IMG_3549](https://user-images.githubusercontent.com/43042045/156999190-254c4acc-f4d3-4510-8868-2ecf3a14af72.JPG)

**Comment by luis901101** [comment]
Hi @felixcary.
For **"manually scan"** also known in documentation as **`softwareTrigger`** I just released a new version [v3.2.0+13](https://pub.dev/packages/honeywell_scanner/versions/3.2.0+13) which allows to `startScanning()` and `stopScanning()`, you can check the plugin readme for more info and the example project for better understanding.
Regarding the **"Continuos"** scanning mode, that is a property configuration which is already available by setting the corresponding property. For advanced use of this plugin it's mandatory to read the official Honeywell documentation, in the readme I point how to get the official documentation and to make devs lives easier I also included in the source code of this plugin a `doc` folder with the necessary info to know each property key and value.

From `doc` folder, file `BarcodeReader.html`, you can find:
```html
PROPERTY_TRIGGER_SCAN_MODE
public static final java.lang.String PROPERTY_TRIGGER_SCAN_MODE
Trigger scan mode. This setting determines what the behavior will be when the trigger is pressed.
```
and
```html
TRIGGER_SCAN_MODE_CONTINUOUS
public static final java.lang.String TRIGGER_SCAN_MODE_CONTINUOUS
Continuous. When this setting is applied, pressing the trigger button will allow to scan multiple barcodes, however duplicate barcodes are not scanned.
```

Then with the names of the properties you can go to `doc` folder, file `BarcodeReaderProperties.java` and find:
```java
public static final String PROPERTY_TRIGGER_SCAN_MODE = "TRIG_SCAN_MODE";
public static final String TRIGGER_SCAN_MODE_CONTINUOUS = "continuous";
```
And finally you can set the key,value pair you need to set the property:

```dart
honeywellScanner.setProperties({
   'TRIG_SCAN_MODE': 'continuous',
});
```

**Note:** take into account that `.setProperties({...})` it's a SET function, so you should set all properties you need at once.

**Comment by felixcary** [comment]
Hi @luis901101, thank you so much for your help! I'll try the new version and also property configuration.
